epub33/core/index.html

```diff
@@ -1340,58 +1337,55 @@ <h5>Manifest Fallbacks</h5>
 						<p>Reading System support for manifest fallbacks is poor and should not be relied upon to create
 							interoperable content. In most cases, if a Reading System does not support a format, it will
 							not use its fallback. This can lead to unexpected failures.</p>
-
-						<p>EPUB Creators should use the intrinsic fallback capabilities of HTML and SVG to provide
-							fallback content whenever possible, and avoid using <a>Foreign Content Documents</a> in the
-							spine.</p>
```

Comment:
I understand the reason for the caution but, spec-wise, it looks a bit strange to start a section with essentially saying "ignore this section". This may raise some question like "why do you have this section in the first place?" or "why is it not deprecated instead?". We know the answer but, nevertheless...

I think it looks somewhat less awkward if that caution is moved to the end of the section.

Reply:
Or should we drop this one for now?

This is essentially pre-empting the "under-implemented" label which is going to tell people exactly the same thing. The caution about not needing to use manifest fallbacks for content anymore is a legitimate separate issue, but perhaps that only needs to be a note given it will also fall into the same general lack of support bucket.

Reply:
Yep, I think it is better to drop it for now. 

We will have to think about how we will, editorially, mark up under-implemented features... maybe something more than just a parenthetical word would be needed, We will see.